src/cdk-experimental/column-resize/column-resize-directives/column-resize-flex.ts

```diff
@@ -27,7 +27,7 @@ import {FLEX_PROVIDERS} from './constants';
 export class CdkColumnResizeFlex extends ColumnResize {
   constructor(
       readonly columnResizeNotifier: ColumnResizeNotifier,
-      protected readonly elementRef: ElementRef,
+      readonly elementRef: ElementRef,
```

Comment:
Can we type all of these `ElementRef` injections to `ElementRef<HTMLElement>` for better type safety?